game/csgo/cfg/SharpTimer/config.cfg

```diff
@@ -21,7 +21,7 @@ sharptimer_custom_map_cfgs_enabled                  true
 sharptimer_command_spam_cooldown                    0.5                                     // Defines the time in seconds between commands can be called. Default value: 0.5
 sharptimer_remove_legs                              true                                    // Whether Legs should be removed or not. Default value: true
 sharptimer_remove_collision                         true                                    // Whether player collision should be removed or not. Default value: true
-sharptimer_remove_damage                            true                                    // Whether dealing damage should be disabled or not (will crash with cs2fixes since they hook/detour damage regardless of you using their feature). Default value: true
+sharptimer_remove_damage                            false                                    // Whether dealing damage should be disabled or not (will crash with cs2fixes since they hook/detour damage regardless of you using their feature). Default value: true

```

Comment:
Changing the valve to false will reactivate the damage feature.
However,due to courses maps design(Triggers hurt on the start point after the game started for about 60s),enabling damage again will cause players on the start point got killed by trigger hurts over and over again.
Setting a spawnpos on the map could be necessary.